webpack/react_app/components/TargetingHosts/TargetingHostsPage.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import SearchBar from 'foremanReact/components/SearchBar';
+import Pagination from 'foremanReact/components/Pagination/PaginationWrapper';
+import { getControllerSearchProps } from 'foremanReact/constants';
+
+import TargetingHosts from './TargetingHosts';
+import './TargetingHostsPage.scss';
+
+const TargetingHostsPage = ({
+  handleSearch,
+  searchQuery,
+  status,
+  items,
+  autoRefresh,
+  totalHosts,
+  pagination,
+  handlePagination,
+}) => (
+  <div id="targeting_hosts">
+    <div className="row">
+      <div className="title_filter col-md-6">
+        <SearchBar
+          onSearch={query => handleSearch(query)}
+          data={{
+            ...getControllerSearchProps('hosts'),
+            autocomplete: {
+              id: 'targeting_hosts_search',
+              searchQuery,
+              url: '/hosts/auto_complete_search',
+              useKeyShortcuts: true,
+            },
+            bookmarks: {},
```

Comment:
We don't want bookmarks in this search?

Reply:
Not in this PR, because if you enable bookmarks they reload the page with new URL, but `TargetingHosts` component ignores params from URL so nothing will change, just page reload.

Since they weren't there before I don't think it's a big issue.

Reply:
Yeah makes sense